enqueue.js

```diff
@@ -0,0 +1,36 @@
+const { SQSClient, SendMessageCommand } = require('@aws-sdk/client-sqs');
+require('dotenv').config();
```

Comment:
nit: sort the lines for import

Reply:
Ok, that makes sense since, to me it makes intuitive sense to place the 'dotenv' import first since in provides environment variables that are consumed by the SQS client so I can change that. Or were you referring to a different type of sort?